api/views.py

```diff
@@ -115,11 +121,11 @@ def get_queryset(self):
         all_users = super().get_queryset()
         # Users with can_read_all_user_data permissions can view all active users via the API
         if self.request.user.has_perm('accounts.can_read_all_user_data'):
-            return all_users.filter(is_active=True)
+            return all_users.defer('middle_name').filter(is_active=True)
```

Comment:
Thanks for catching that - that was from playing around with defer initially to see if that would work to omit usernames from responses. (Was deferring middle_name in case username wasn't dropped because it's used as an index.) Removed now!